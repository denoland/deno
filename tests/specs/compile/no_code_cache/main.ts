import { join } from "jsr:@std/url@0.220/join";

console.log(join);
