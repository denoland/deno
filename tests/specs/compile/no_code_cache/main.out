[WILDCARD]Code cache disabled.[WILDCARD]